yjit/src/yjit.rs

```diff
@@ -134,6 +134,13 @@ pub extern "C" fn rb_yjit_iseq_gen_entry_point(iseq: IseqPtr, ec: EcPtr, jit_exc
         return std::ptr::null();
     }
 
+    // If this is a large application (has very many ISEQs), switch to
+    // using the call threshold for large applications after this entry point
+    use crate::stats::rb_yjit_live_iseq_count;
+    if unsafe { rb_yjit_live_iseq_count } > LARGE_ISEQ_COUNT {
+        unsafe { rb_yjit_call_threshold = LARGE_CALL_THRESHOLD; };
```

Comment:
I think we'd like to skip this when `--yjit-call-threshold` is explicitly specified. SFR uses 1000, but the current patch would decrease it to hard-coded 120.

Reply:
made a suggested patch https://github.com/ruby/ruby/pull/8839/files#r1382161082